Libraries/Network/NetInfo.js

```diff
@@ -168,6 +168,9 @@ const _isConnectedSubscriptions = new Map();
  * ```
  */
 const NetInfo = {
+  /*
+   * Invokes the handler whenever the network info changes. The handler is passed with any of the connectivity types listed above.
```

Comment:
@janicduplessis from the contributing guide, it says

> Do not wrap lines at 80 characters - configure your editor to soft-wrap when editing documentation.

Did I understand incorrectly?